functions/submarine/fn_submarineSelectPos.sqf

```diff
@@ -0,0 +1,25 @@
+// Mandatory to get suspension allowed
+_this spawn {
+	params ["_units", "_pos", "_alt", "_shift"];
+
+	// Map click may happen but not linked to this module
+	if!(player getVariable ["submarine_flag", false]) exitWith {};
+
+	// Check if there is water here
+	_maxDepth = getTerrainHeightASL _pos;
+	if(_maxDepth > -25 and worldName != "VR") exitWith { openMap false; ["SubmarineFail"] call BIS_fnc_showNotification};
+
+	// TODO : check if there is players around the sub, both at actual and targeted coordinates
```

Comment:
Create an issue for that ?

Reply:
Done https://github.com/team-gsri/OpexTemplate/issues/9